### PR TITLE
todo does not output anything when their are no list.(for #118)

### PR DESCRIPTION
**Pull request by AnubhaAgrawal**
todo does not output anything when their are no list.

### CODE REVIEW

**Comment by untitaker** [comment]
@AnubhaAgrawal Thanks, this looks good. Perhaps you could add a bit more text that explains the part that you found not self-explanatory. A few questions arise: 1.) How do I create a list? 2.) Where do I create it?

File: todoman/cli.py [hunk 1]
```diff
@@ -143,6 +143,11 @@ def cli(ctx, color, porcelain):
     if len(paths) == 0:
         click.echo("No lists found.\n")
         click.echo(config["main"]["path"])
+        click.echo("goto ~/.local/share/calendars")
```
Comment (untitaker) [inline todoman/cli.py]: `~/.local/share/calendars` is the default path, but the user may have configured another one. This is what `config["main"]["path"]` is for.

Something like: `"No lists found matching {}, create a directory for a new list".format(config["main"]["path"]`

**Comment by untitaker** [comment]
Also please add a testcase and changelog entry.

You can prepare a testcase using the `runner` fixture [as shown here](https://github.com/pimutils/todoman/blob/master/tests/test_basic.py#L14). However, by default there is already a list created, so you also need to do `tmpdir.join('default').remove()` before testing.

File: tests/test_basic.py [hunk 2]
```diff
@@ -425,5 +426,17 @@ def test_edit(runner, default_database):
     assert todo.due == datetime.datetime(2017, 2, 1, tzinfo=tzlocal())
     assert todo.summary == 'Eat paint'
 
+
+def test_empty_list(tmpdir, runner, create):
+    [item.remove()
```
Comment (untitaker) [inline tests/test_basic.py]: Please use a normal for-loop here.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -21,7 +21,7 @@ v3.0.0
 * Make the output of move, delete, copy and flush more consistent with
   everything else.
 * Porcelain now outputs proper JSON, rather than one-JSON-per-line.
-
+* todo does not output anything when their are no list.
```
Comment (hobarrera) [inline CHANGELOG.rst]: Please rewrite this. This changelog should explain in simple term to users what changed. This explains what the bug was, not what we changed.

File: todoman/cli.py [hunk 4]
```diff
@@ -144,6 +144,11 @@ def cli(ctx, color, porcelain):
         path for path in glob.iglob(expanduser(config["main"]["path"]))
         if isdir(path)
     ]
+    if len(paths) == 0:
+        click.echo("No lists found matching {}, "
```
Comment (hobarrera) [inline todoman/cli.py]: This string can fit into a single line (78 characters), no need to split it.

Comment (untitaker) [inline todoman/cli.py]: No, it doesn't fit. The string will wrap on the last word.

Comment (hobarrera) [inline todoman/cli.py]:
```
In [1]: len('    click.echo("No lists found matching {}, create a directory for a new list"')
Out[1]: 78
```

Comment (untitaker) [inline todoman/cli.py]: You're missing a level of indentation.

**Comment by hobarrera** [comment]
Also, not strictly related to this PR, @AnubhaAgrawal, can you check your system's time? Your commits have dates from the future (and github doesn't seem to mind about that).

**Comment by AnubhaAgrawal** [comment]
Now, I correct my system's time, please tell me what should I do now?

**Comment by untitaker** [comment]
LGTM

**Comment by untitaker** [comment]
Thanks @AnubhaAgrawal for your PR!

**Pull request merged**